integration-tests/conftest.py

```diff
@@ -2,6 +2,8 @@
 import subprocess
 import time
 
+HOST_DETAILS: str = "/var/lib/insights/host-details.json"
```

Comment:
Should this use a hard-coded prefix? Or should the build tool construct this path from its prefix and [other default build directories](https://mesonbuild.com/Builtin-options.html#directories)? I would imagine that because we're testing software that's immediately been built, we could incorporate these files into the build system too?

Reply:
do you mean something like this:   HOST_DETAILS = os.path.join(INSIGHTS_LIB_DIR, 'host-details.json')?

Reply:
I can imagine this scenario:
builder of the package creates properties.toml file with varibles aka: build.lib, buils.sysconfig, ....

testware (uses DYNACONF to load properties) loads the properties.
testware offers the properties to tests to create HOST_DETAILS, ... properties.

is it the case you think @subpop ?